components/fieldgroup/stories/fieldgroup.test.js

```diff
@@ -7,47 +6,51 @@ export const FieldGroupSet = Variants({
 	testData: [
 		{
 			testHeading: "Default",
+			inputType: "radio",
```

Comment:
Because `inputType: "radio` is the default arg for the Template, we don't actually need this for any of the radio tests! 🎉